File: Assets/VirtualCity/ProtoDefine/Java/system/RankInfo.java
package com.kingston.jforgame.server.game.system.message;

import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;

public class RankInfo {
    @Protobuf(order = 1)
    private Long accountId;
    @Protobuf(order = 2)
    private double info;
    @Protobuf(order = 3)
    private Long modelId;
    @Protobuf(order = 4)
    private String name;

    public Long getAccountId() {
        return accountId;
    }

    public void setAccountId(Long accountId) {
        this.accountId = accountId;
    }

    public double getInfo() {
        return info;
    }

    public void setInfo(double info) {
        this.info = info;
    }

    public Long getModelId() {
        return modelId;
    }

    public void setModelId(Long modelId) {
        this.modelId = modelId;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}

File: Assets/VirtualCity/ProtoDefine/RankInfo.cs
using System.Collections.Generic;
using ProtoBuf;
namespace ProtoDefine {
[ProtoContract]


public class RankInfo {
[ProtoMember(1)]
    public long? accountId;
[ProtoMember(2)]
    public double info;
[ProtoMember(3)]
    public long? modelId;
[ProtoMember(4)]
    public string name;

    public long? getAccountId() {
        return accountId;
    }

    public void setAccountId(long? accountId) {
        this.accountId = accountId;
    }

    public double getInfo() {
        return info;
    }

    public void setInfo(double info) {
        this.info = info;
    }

    public long? getModelId() {
        return modelId;
    }

    public void setModelId(long? modelId) {
        this.modelId = modelId;
    }

    public string getName() {
        return name;
    }

    public void setName(string name) {
        this.name = name;
    }
}
}